### PR TITLE
Feature/UI improvements

### PR DESCRIPTION
**Pull request by MikaelNordberg**
HTML changes for improved styling possibilities

### CODE REVIEW

**Review by likp — APPROVED** [review]
Looks great!

File: PXWeb/Presentation.master [hunk 1]
```diff
@@ -23,12 +23,16 @@
         <div id="subheader">
             <pxwebCustomControl:HeadingLabel ID="lblSubHeader" runat="server" Text="<%$ PxString: PxWebSubHeaderResult%>"></pxwebCustomControl:HeadingLabel>
         </div>
-        <div>
+        <div class="menu-item">
```
Comment (MikaelNordberg) [inline PXWeb/Presentation.master:26]: Change to burger-menu-item

**Review by JohannesFinsveen — APPROVED** [review]
Approved! :-)

**Review by likp — APPROVED** [review]
Looks even better

File: PXWeb/Presentation.master (hunk 1 above)
Comment (MikaelNordberg) [reply PXWeb/Presentation.master]: Name of css class has been changed

**Pull request merged**